### ISSUE TITLE
[Event Request] Page 9248 - OnBeforeEnter_BaseQty/OnBeforeProdForecastQtyBase_OnValidate

### ISSUE THREAD

**Issue by auzhhv** [microsoft/ALAppExtensions#4413]
Can you add events in page 9248 "Demand Forecast Matrix"? 
Event OnBeforeEnter_BaseQty in function Enter_BaseQty, and event OnBeforeProdForecastQtyBase_OnValidate in function ProdForecastQtyBase_OnValidate
With IncludeSender = Yes and GlobalVarAccess = Yes, for both.

Enter_BaseQty:
![image](https://user-images.githubusercontent.com/46031720/65953179-bb6dfc80-e443-11e9-962c-e1696e7b3afb.png)

ProdForecastQtyBase_OnValidate:
![image](https://user-images.githubusercontent.com/46031720/65953194-c4f76480-e443-11e9-9260-522c93a85bc5.png)

Events:
![image](https://user-images.githubusercontent.com/46031720/65953216-ce80cc80-e443-11e9-898d-74d0bafc1e14.png)

**Comment by AlexanderYakunin** [comment]
No, we cannot add events with GlobalVarAccess. This property discontinued.

**Comment by bc-ghost** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. We will update this issue with information about availability.
